themes/bootstrap3/templates/RecordTab/similaritemscarousel.phtml

```diff
@@ -53,6 +52,7 @@
 <?php endif; ?>
 <?php
   $script = <<<JS
+loadCovers();
```

Comment:
This throws an undefined function error when AJAX covers are disabled. I think we need to check if it's defined before we call it. Also, is there a risk that this will cause some images on the page to double-load, because the function was previously called through the document ready event? If there's not already a mechanism in place to protect against double-loading, we might want to add one (perhaps a data attribute on the image, or something similar).

Reply:
As I remember, it needed to be called secondly when a tab is loaded by ajax... But I add a check if ajax covers are enabled.

 I agree we should prevent double loading of images, I'll fix it, but probably not till monday...

Reply:
I've added check for ajaxCoversEnabled in similar items tab

Reply:
Thanks!